packages/angular_devkit/build_angular/src/browser/schema.json

```diff
@@ -380,6 +380,14 @@
       "type": "boolean",
       "description": "Concatenate modules with Rollup before bundling them with Webpack.",
       "default": false
+    },
+    "allowedCommonJsDepedencies": {
+      "description": "A list of CommonJS packages or entry-points that are allowed to be used without a built time warning.",
```

Comment:
Nit: I'd remove "or entry-points " here, because we mostly mean polyfills and main when talking about entry points.

Reply:
Updated!